README.md

```diff
@@ -143,6 +143,6 @@ In order to produce a uniquely identifiable distribution:
 Feedstock Maintainers
 =====================
 
-* [@elichad](https://github.com/elichad/)
-* [@stur86](https://github.com/stur86/)
+* [@leandro-liborio](https://github.com/leandro-liborio/)
+* [@patrick-austin](https://github.com/patrick-austin/)
 
```

Comment:
```suggestion
* [@joelvdavies](https://github.com/joelvdavies/)

```
Should you also be in this list, given you're listed in the other two files, or was this a deliberate omission?

Reply:
Deliberate - I think its unlikely I will be doing anything with this but thought it best to leave the others for now.

Reply:
Fair enough - don't know how likely it is I will either (having said that I'm sure I've now jinxed myself), but someone should be listed I suppose.